src/distribution/value.coffee

```diff
@@ -125,10 +130,18 @@ module.exports = do ->
     switch choice_index
 
       when FIRST_CHOICE
-        return domain_create_value domain_get_value_of_first_contained_value_in_list fdvar.dom, list
+        v = domain_get_value_of_first_contained_value_in_list fdvar.dom, list
```

Comment:
Pretty long function name, too long to my taste but as the advantage of being pretty clear.